CHANGELOG.md

```diff
@@ -17,6 +17,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - `Edit on Github` button now hidden for plugins.
 - Doctests now run via unittests
+- base classes now use `CONFIG` if it has a default for every field and `config` is `None`
```

Comment:
```suggestion
- Classes now use `CONFIG` if it has a default for every field and `config` is `None`
```

This is applicable to all classes since you've changed the base class